gaphor/ui/filemanager.py

```diff
@@ -177,11 +180,14 @@ def save(self, filename):
 
             self.filename = filename
             self.event_manager.handle(FileSaved(self, filename))
-        except Exception:
+        except Exception as e:
             error_handler(
-                message=gettext("Error while saving model to file {filename}").format(
+                message=gettext("Unable to save model “{filename}”.").format(
```

Comment:
Should we decode the filename to remove the `b' '` wrapping?

Reply:
It looks fine with me (see screenshot above).

Reply:
It looks good now, thanks!